Simplenote/build.gradle

```diff
@@ -65,10 +72,10 @@ buildscript {
 dependencies {
     testImplementation 'junit:junit:4.12'
     testImplementation 'org.mockito:mockito-core:2.8.47'
-    androidTestImplementation 'androidx.test:core:1.0.0'
+    androidTestImplementation 'androidx.test:core:1.2.0'
     androidTestImplementation 'androidx.annotation:annotation:1.0.0'
-    androidTestImplementation 'androidx.test:runner:1.1.0'
-    androidTestImplementation 'androidx.test:rules:1.1.0'
+    androidTestImplementation 'androidx.test:runner:1.2.0'
+    androidTestImplementation 'androidx.test:rules:1.2.0'
```

Comment:
Question (❓): Why did you choose to update to these libraries to `1.2.0` and not the actual latest version (which is the `1.3.0`)?

Reply:
I was careful on bumping libraries and I did not want to bump it to far from the current version. But `1.3.0` (I did not check `1.3.0` was the last one) is not a mayor version, thus I think it is OK to bump it to `1.3.0`.

Reply:
👋 @dandominguez04 !

Yeap, I agree, I think since we are bumping the versions, bumping up to the latest might be beneficial for us. And as you said, since this is not a major version update, I too think we are okay.